hasher-matcher-actioner/hmalib/common/models/bank.py

```diff
@@ -127,21 +139,74 @@ class BankMember(DynamoDBItem):
     video, photo that a partner wants to match against.
     """
 
+    BANK_MEMBER_ID_PREFIX = "member#"
+
     bank_id: str
     bank_member_id: str
```

Comment:
I'm afraid I do not follow. ContentId is part of the pipeline models which are completely independent of the bank models. 

Even if a pipeline object is used to add to a bank, banks will copy it into its own s3 bucket and use that as a reference. This is because the retention could be configurable on the pipeline and bank data / storage. Banks do not know about pipeline and pipeline does not know about banks. They only intersect at the index.

Reply:
Yeah. We could abuse 'notes' there as well. But avoid a strong link as much as possible.